content/docs/gke/custom-domain.md

```diff
@@ -4,54 +4,62 @@ description = "Using a custom domain with Kubeflow on GKE"
 weight = 30
 +++
 
-{{% needs-update %}}
-
-This guide describes how to use a custom domain with Kubeflow on Google Cloud 
+This guide describes how to use a custom domain with Kubeflow on Google Cloud
 Platform (GCP).
 
 ## Before you start
 
 This guide assumes you have already set up Kubeflow on GCP. If you haven't done
-so, follow the guide to 
+so, follow the guide to
 [getting started with Kubeflow on GCP](/docs/gke/deploy/).
 
 ## Using your own domain
 
-If you want to use your own domain instead of **${KF_NAME}.endpoints.${PROJECT}.cloud.goog**, follow these instructions:
-
-1. Remove the `cloud-endpoints` component:
+If you want to use your own domain instead of **${KF_NAME}.endpoints.${PROJECT}.cloud.goog**, follow these instructions after running `kfctl build`:
 
+1. Replace `hostname` in `kustomize/iap-ingress/base/params.env` with your own domain.
     ```
-    cd ${KF_DIR}/kustomize
-    kubectl delete -f cloud-endpoints.yaml
+    ...
+    hostname=<enter your domain here>
+    ingressName=envoy-ingress
+    ipName=kf-test-ip
+    oauthSecretName=kubeflow-oauth
+    ...
     ```
 
-1. Set the domain for your ingress to be the fully qualified domain name:
+1. Apply the changes.
+
+    Using `kfctl`.
 
     ```
-    cd ${KF_DIR}/kustomize
-    gvim iap-ingress.yaml    # Or basic-auth-ingress.yaml
+    kfctl apply -V -f ${CONFIG_FILE}
     ```
 
-   * Find and replace the value for the hostname:
+    Or using `kubectl`.
 
-     ```
-     data:
-       hostname: <enter your endpoint here>
-     ```
+    ```
+    kubectl apply -k kustomize/iap-ingress
+    ```
 
-   * Apply the changes:
+1. Remove the `cloud-endpoints` component.
+    ```
+    kubectl delete -k kustomize/cloud-endpoints
+    ```
 
-     ```
-     kubectl apply -f iap-ingress.yaml
-     ```
+    Delete the endpoint created by the `cloud-endpoints-controller`, if any.
+    ```
+    gcloud endpoints services delete ${KF_NAME}.endpoints.${PROJECT}.cloud.goog
+    ```
 
-1. Get the address of the static IP address created:
+1. Check Ingress to verify that your domain was properly configured.
+    ```
+    kubectl -n istio-system describe ingresses
+    ```
 
+1. Get the address of the static IP address created.
     ```
     IPNAME=${KF_NAME}-ip
-    gcloud --project=${PROJECT} compute addresses describe --global ${IPNAME}
+    gcloud compute addresses describe ${IPNAME} --global --format="value(address)"
     ```
 
-1. Use your DNS provider to map the fully qualified domain specified in the first step to the IP address reserved:
-   in GCP.
+1. Use your DNS provider to map the fully qualified domain specified in the first step to the IP address reserved in GCP.
```

Comment:
nit: maybe just say "...to the above IP address" instead of "...to the IP address reserved in GCP" to make it more clear.